v3/src/components/graph/components/inspector-panel/hide-show-menu-list.tsx

```diff
@@ -78,6 +78,19 @@ export const HideShowMenuList = observer(function HideShowMenuList({tile}: IProp
     )
   }
 
+  const handleMeasuresForSelectionChange = () => {
+    const undoStringKey = graphModel?.showMeasuresForSelection
+                            ? "DG.Undo.disableMeasuresForSelection"
+                            : "DG.Undo.enableMeasuresForSelection"
+    const redoStringKey = graphModel?.showMeasuresForSelection
+                            ? "DG.Redo.disableMeasuresForSelection"
+                            : "DG.Redo.enableMeasuresForSelection"
```

Comment:
Could do something like this to avoid testing `showMeasuresForSelection` twice:
```suggestion
  const [undoStringKey, redoStringKey] = graphModel?.showMeasuresForSelection
                            ? ["DG.Undo.disableMeasuresForSelection", "DG.Redo.disableMeasuresForSelection"]
                            : ["DG.Undo.enableMeasuresForSelection", "DG.Redo.enableMeasuresForSelection"]
```